api/src/auth/drivers/openid.ts

```diff
@@ -178,6 +197,16 @@ export class OpenIDAuthDriver extends LocalAuthDriver {
 			throw handleError(e);
 		}
 
+		let role = this.config['defaultRoleId'];
+
+		// Overwrite default role if user is member of a group specified in roleMap
+		for (const key in this.roleMap) {
+			if ((userInfo['groups'] as Array<string>).includes(key)) {
```

Comment:
This isn't a guaranteed return value for all providers and could therefore fail.

Reply:
The name of the OIDC group claim can now be configured using AUTH_<PROVIDER>_GROUP_CLAIM_NAME. It defaults to 'groups'.